conda_build/build.py

```diff
@@ -1441,13 +1444,14 @@ def create_info_files(m, replacements, files, prefix):
 
 
 def get_short_path(m, target_file):
+    if m.python_version_independent:
+        if target_file.find("site-packages") >= 0:
+            return target_file[target_file.find("site-packages") :]
```

Comment:
```suggestion
        if (site_packages_idx := target_file.find("site-packages")) >= 0:
            return target_file[site_packages_idx:]
```

Perfect case for walrus!